packages/zowe-explorer/CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
-- Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish. Download the respective language pack and switch.
-- Localization of strings within the webviews. [#2983](https://github.com/zowe/zowe-explorer-vscode/issues/2983)
-- Added expired JSON web token detection for profiles in each tree view (Data Sets, USS, Jobs). When a user performs a search on a profile, they are prompted to log in if their token expired. [#3175](https://github.com/zowe/zowe-explorer-vscode/issues/3175)
```

Comment:
should this item stay for 3.1 under TBD for now?

Reply:
For this PR, I did a `git checkout` of the changelogs on the v3.0.2 tag. The contents should match the changelogs on 3.0.2, but with the TBD release headers added above:

https://github.com/zowe/zowe-explorer-vscode/blob/v3.0.2/packages/zowe-explorer/CHANGELOG.md#302

Reply:
Should be resolved in 7f67562 👍